python_and_git_6_7_logic/task_1_num.py

```diff
@@ -0,0 +1,19 @@
+print()
+print("First version Code")
+nx = int(input ("Enter a number: "))
+if nx < 10:
+    print("The number ", nx, " is less than 10")
+elif nx > 20:
+    print("The number ", nx, " is greater than 20")
+else:
+    print("The number ", nx, " is between 10 and 20")
+print()
+print("Second version Code")
+ny = int(input ("Enter a number: "))
+if 20 >= ny >= 10:
+    print("The number ", ny, " is between 10 and 20")
+elif ny > 20:
+    print("The ", ny, " is greater than 20")
+else:
+    print("The number ", ny, " is less than 10")
+print("Thanks")
```

Comment:
Make second version as main version :)
You don't need versions of programs that override each other

```suggestion
ny = int(input ("Enter a number: "))
if 20 >= ny >= 10:
    print("The number ", ny, " is between 10 and 20")
elif ny > 20:
    print("The ", ny, " is greater than 20")
else:
    print("The number ", ny, " is less than 10")
print("Thanks")
```